test/extended/image_ecosystem/plugin.go

```diff
@@ -414,18 +417,46 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 		g.By("kick off the build for the jenkins ephermeral and application templates")
 		tag := []string{"openshift/jenkins-plugin-snapshot-test:latest"}
 		hexIDs, err := exutil.DumpAndReturnTagging(tag)
-		var jenkinsEphemeralPath string
-		var testingSnapshot bool
-		if len(hexIDs) > 0 && err == nil {
-			// found an openshift pipeline plugin test image, must be testing a proposed change to the plugin
-			jenkinsEphemeralPath = exutil.FixturePath("testdata", "jenkins-ephemeral-template-test-new-plugin.json")
-			testingSnapshot = true
+
+		// If the user has expressed an interest in local plugin testing by setting the
+		// SNAPSHOT_JENKINS_IMAGE environment variable, try to use the local image. Inform them
+		// either about which image is being used in case their test fails.
+		snapshotImagePresent := len(hexIDs) > 0 && err == nil
+		useSnapshotImage := os.Getenv(useLocalPluginSnapshotEnvVarName) != "" && snapshotImagePresent
+
+		//TODO disabling oauth until we can update getAdminPassword path to handle oauth (perhaps borrow from oauth integration tests)
+		newAppArgs := []string{exutil.FixturePath("testdata", "jenkins-ephemeral-template-no-oauth.json")}
```

Comment:
Arguably orthogonal to your change, but if we could somehow copy `examples/jenkins/jenkins-ephemeral-template.json` from the origin source tree to a temporary location (the forcepull.go tests do something similar if I remember correctly), and somehow find this snippet in the template:

```
                  {
                    "name": "OPENSHIFT_ENABLE_OAUTH",
                    "value": "true"
                  },
```
and change the value to `false`, we could in fact use the official example template while still disabling oauth.

Assuming this is sufficiently *simple* (certainly *simpler* I think than handling oauth flows in our extended tests) of course.

Perhaps to be done in a separate PR if we want to get this change in quickly.